spec.yaml

```diff
@@ -198,6 +199,176 @@ paths:
+                            "IsInvalidFloat": false
+                            },
+                          }
+                        }
+                      },
+                      "HyperParamsByPath": {
+                        "params.yml": [
+                          {
+                            "ExperimentID": 2301,
+                            "Name": "tfidf__max_df",
+                            "Value": "1",
+                            "Path": "params.yml"
+                          }, 
+                        ]
+                      },
+                      "Labels": [],
+                      "Errors": [],
+                      "Source": "git",
+                      "Group": ""
+                    },
+                  }
+                }
+  /repos/{owner}/{repo}/experiments/metricsForExperiments:
+    post:
+      tags:
+        - Experiments
+      summary: Get experiment metrics for experiment
+      operationId: getExperimentMetrics
+      parameters:
+      - $ref: './components/parameters/Owner.yaml'
+      - $ref: './components/parameters/Repository.yaml'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: './components/schemas/ExperimentKeys.yaml'
+      responses:
+        '200':
+          description: Retrieved experiments metrics
+          content:
+            application/json:
+              example:
+                {
+                  "2306": {
+                    "metrics.csv": {
+                    "test__accuracy": [
+                      {
+                      "Value": 0.91712,
+                      "Timestamp": 1595149274803,
+                      "Step": 1,
+                      "ValueStr": "0.91712"
+                      }
+                    ],
+                    "train__roc_auc": [
+                      {
+                      "Value": 0.9997316690244479,
+                      "Timestamp": 1595149273838,
+                      "Step": 1,
+                      "ValueStr": "0.9997316690244479"
+                      }
+                    ]
+                    }
+                  }
+                }
+  /repos/{owner}/{repo}/experiments/experiment/{experimentKey}:
+    post:
+      tags:
+        - Experiments
+      summary: Set name for experiment
+      description: Set a name for an experiment by experiment key
+      operationId: setExperimentName
+      parameters:
+      - $ref: './components/parameters/Owner.yaml'
+      - $ref: './components/parameters/Repository.yaml'
+      - $ref: './components/parameters/ExperimentKey.yaml'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: './components/schemas/ExperimentNamePost.yaml'
+      responses:
+        '200':
+          description: Set name for experiment successfully
+          content:
+            application/json:
+              example: "Awesome New Experiment Name"
```

Comment:
This endpoint description is now inaccurate - it's now a general puprpose "edit experiment" API endpoint, that only allows editing the name for now